CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-authn-k8s-client#206](https://github.com/cyberark/conjur-authn-k8s-client/issues/206)
 - Improve the error message raised when the username doesn't include the `host/` prefix
   [cyberark/conjur-authn-k8s-client#212](https://github.com/cyberark/conjur-authn-k8s-client/pull/212)
+- The project's version will not present the git commit hash on a fixed versions
```

Comment:
@shulifink can you assist please?

Reply:
I don;t get the format of the change log. Shouldn't it show what changed? it in past tense, or "No longer does this..." or "changed this to this", or "Now you can this and that", or "Fixed this so that blah blah"?

Reply:
So eg  here I would say something like:

Doesn't include git commit hash in printed version for fixed versions

What/who is not including the git command hash?

Reply:
@shulifink @orenbm  thanks for the comments
@shulifink during the build process (while creating the product container) the script checks the commits to identify whether this is a fixed version and in that case it will not concatenate the git commit hash to its version.
changing the description to your suggestion

Reply:
I underdatnd. Thanks for the explanation and for pointing out to me that I can look in the original ticket to see what was changed. This helped me understand what I was reviewing :)

So another question: What is a  "fixed version" ? Is this "fixed" in terms of the version was not working and now it's been fixed, or is this "fixed" as in it is a "static" version?

Reply:
@shulifink the later... it refers to a release version